doc/decisions/decision_process.md

```diff
@@ -24,24 +26,40 @@ Substantial decisions must be made in a transparent and participative way.
   - Rebase only if the decision was already accepted and has a merge conflict.
 - For reviewers:
   - Prefer to directly give suggestions how to change sentences.
-  - General questions should be asked in the root of "Conversation" and not at random sentences in the review.
+  - General questions should be asked in the root of "Conversation" and not at vaguely related sentences in the review.
+- Decision PRs do not significantly change anything but one decision.
```

Comment:
> I would even forbid typo fixes outside the already changed files.

I think at least backlinks should be allowed, otherwise it gets unpractical, see 5b70006d13907b6101ff60cd4eab5ffa0f8163a8